### PR TITLE
add --quiet=true for swag init, make the debug logger quiet.

### PR DESCRIPTION
**Pull request by jixiuf**
**Describe the PR**
Add a `--quiet`=true flag for `swag init`.
make the debug logger quiet.
**Relation issue**
https://github.com/swaggo/swag/issues/1193

### CODE REVIEW

File: cmd/swag/main.go [hunk 1]
```diff
@@ -29,9 +30,14 @@ const (
 	parseDepthFlag       = "parseDepth"
 	instanceNameFlag     = "instanceName"
 	overridesFileFlag    = "overridesFile"
+	quiet                = "quiet"
```
Comment (ubogdan) [inline cmd/swag/main.go:33]: I prefer `-q` flag in favor  of `-quiet`.

Comment (jixiuf) [reply cmd/swag/main.go]: > I prefer the `q` flag in favor of `quiet`. If this flag disables all info output, we may use `s` for silent.

I add a alias `q`, is that ok for you?
https://github.com/swaggo/swag/blob/quiet_flag/cmd/swag/main.go#L39

Comment (ubogdan) [reply cmd/swag/main.go]: ok

File: cmd/swag/main.go [hunk 2]
```diff
@@ -30,9 +31,15 @@ const (
 	instanceNameFlag     = "instanceName"
 	overridesFileFlag    = "overridesFile"
 	parseGoListFlag      = "parseGoList"
+	quietFlag            = "quiet"
 )
 
 var initFlags = []cli.Flag{
+	&cli.BoolFlag{
+		Name:    quietFlag,
+		Aliases: []string{q},
```
Comment (ubogdan) [inline cmd/swag/main.go:40]: You forgot to put the quotes :)
Error: vet: cmd/swag/main.go:40:21: undeclared name: q

**Review by ubogdan — APPROVED** [review]
LGTM

**Comment by ubogdan** [comment]
@jixiuf Thanks for your contribution.

**Pull request merged**